src/enums.h

```diff
@@ -413,7 +413,15 @@ enum PlayerSex_t : uint8_t
 
 enum Vocation_t : uint16_t
 {
-	VOCATION_NONE = 0
+	VOCATION_NONE = 0,
+	VOCATION_SORCERER = 1,
+	VOCATION_DRUID = 2,
+	VOCATION_PALADIN = 3,
+	VOCATION_KNIGHT = 4,
+	VOCATION_MASTER_SORCERER = 5,
+	VOCATION_ELDER_DRUID = 6,
+	VOCATION_ROYAL_PALADIN = 7,
+	VOCATION_ELITE_KNIGHT = 8
 };
```

Comment:
This does not belong here, it should be exclusively in Lua.

Reply:
I figured someone would say that, but I still wanted to do it this way.
I thought that as VOCATION_NONE would already exist, there would be no problems in leaving the other vocations reserved, because there is no problem with that, and maybe it could be used in the not so distant future (maybe with the highscore system).

Reply:
Yeah, but `VOCATION_NONE` is a special case and it still should be removed whenever possible

Highscore system is related to #4152

Reply:
Any suggestions on how and where to store the information in lua?

Reply:
`data/lib/core` and constants.lua or vocation.lua

````
VOCATION_SORCERER = 1
VOCATION_DRUID = 2
VOCATION_PALADIN = 3
VOCATION_KNIGHT = 4
VOCATION_MASTER_SORCERER = 5
VOCATION_ELDER_DRUID = 6
VOCATION_ROYAL_PALADIN = 7
VOCATION_ELITE_KNIGHT = 8
```